stdlib/Random/src/normal.jl

```diff
@@ -35,7 +35,26 @@ julia> randn(rng, ComplexF32, (2, 3))
   0.611224+1.56403im   0.355204-0.365563im  0.0905552+1.31012im
 ```
 """
-@inline randn(rng::AbstractRNG=default_rng()) = _randn(rng, rand(rng, UInt52Raw()))
+@inline function randn(rng::AbstractRNG=default_rng())
+    #=
+    When defining 
+    `@inline randn(rng::AbstractRNG=default_rng()) = _randn(rng, rand(rng, UInt52Raw()))`
+    the function call to `_randn` is currently not inlined, resulting in slightly worse
```

Comment:
I was wondering whether it's really `_randn` which fails to be inlined... Looking rapidly at `@code_typed randn(MersenneTwister())` doesn't show a call to `_randn` if I'm not mistaken. How did you conclude that `randn_` is not inlined?

Reply:
I deduced it from the `@code_llvm` (but I might be wrong here - I do not have much experience with these topics):
In the Julia 1.5 version, there is the following call:
`%30 = call double @j__randn_3665(%jl_value_t* nonnull %0, i64 %29) #0`
In the Julia 1.4 code, this call is missing, but there are ca. 60 additional lines of LLVM code. I suppose this is the inlined function, but am not sure about it.

Reply:
Since `_randn` is defined as calling `randn`, the inliner decides to stop. This is reflected in `code_llvm`, but due to the particular way that `code_typed` works and is cached, it won't be represented there.

Reply:
`randn_unlikely` is the noinline slow path though. If we moved the code we'd have to inline everything?